pkg/project/cloud/Taskfile.yml

```diff
@@ -118,27 +121,29 @@ tasks:
         echo "$PROPS" | sh aemw repl agent setup -P --location "publish" --name "flush"
 
   aem:provision:crx:
-    desc: enable CRX/DE
+    desc: enables CRX/DE on all AEM instances
+    internal: true
     cmds:
       - 'sh aemw osgi config save --pid "org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet" --input-string "alias: /crx/server"'
 
   aem:provision:apm:
-    desc: deploy APM tool
+    desc: deploys APM tool
+    internal: true
```

Comment:
I was thinking about it but for troubleshooting it worth sometimes to not have it internal to allow a standalone cmd execution?